prescriptions/_generic/cuda_no_gpu.yaml

```
@@ -0,0 +1,18 @@
+units:
+  boots:
+  - name: CUDANoGPUBoot
+    type: boot
+    should_include:
+      adviser_pipeline: true
+      runtime_environments:
+        cuda_version: ">=0.0"
+        hardware:
+        - gpu_models: [null]
+    run:
+      stack_info:
+      - type: WARNING
+        message: >-
+          A CUDA is available but no GPU information provided, adjust GPU
+          configuration option to have recommendations suited for hardware
+          available
+        link: no_gpu
```

Should we write 

```suggestion
        link: cuda_no_gpu
```

to keep a consistency with [gpu_no_cuda.yaml](https://github.com/thoth-station/prescriptions/blob/master/prescriptions/_generic/gpu_no_cuda.yaml) ?

---

This one is referring to the following justification: https://thoth-station.ninja/j/no_gpu

---

I see, thanks!